google/auth/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.34.0"
+__version__ = "2.0.0b1"
```

Comment:
I would prefer the version be bumped in a separate release PR  (through `Release-As: 2.0.0b1`)

Reply:
35f4546